clients/usercodedeployment/usercodedeployment-client/src/main/java/Client.java

```diff
@@ -0,0 +1,32 @@
+import com.hazelcast.client.HazelcastClient;
+import com.hazelcast.client.config.ClientConfig;
+import com.hazelcast.client.config.ClientUserCodeDeploymentConfig;
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.core.IMap;
+
+public class Client {
+
+    public static void main(String[] args) throws InterruptedException {
+        ClientConfig clientConfig = new ClientConfig();
+        ClientUserCodeDeploymentConfig clientUserCodeDeploymentConfig = new ClientUserCodeDeploymentConfig();
+        clientUserCodeDeploymentConfig.addClass("IncrementingEntryProcessor");
+        clientConfig.setUserCodeDeploymentConfig(clientUserCodeDeploymentConfig.setEnabled(true));
```

Comment:
you may put setEnabled on a separate line to make it more obvious for the user since this is a sample code.